ldap2pg/ldap.py

```diff
@@ -203,6 +203,10 @@ def connect(**kw):
 
     conn = LDAPLogger(conn)
 
+    if not options.get('REFERRALS'):
+        logger.debug("HOTFIX https://stackoverflow.com/questions/18793040/python-ldap-not-able-to-bind-successfully")
```

Comment:
That should be a comment for developers, not a log message for users. Don't you think so ?